frontend/components/AgentStaking/AgentStaking.tsx

```diff
@@ -78,55 +72,20 @@ type SelectionTabsProps = {
   setCurrentTab: (value: SelectionTabValue) => void;
 };
 
-type SelectionTabProps = {
-  label: string;
-  value: string;
-  isSelected: boolean;
-};
-
-const SelectionTab = ({ label, value, isSelected }: SelectionTabProps) => {
```

Comment:
updated Segmented component as needed, so there's no need to customise like this